Project.toml

```diff
@@ -17,7 +17,7 @@ CondaPkg = "0.2"
 MLJ = "0.17, 0.18"
 MLJBase = "0.19, 0.20"
```

Comment:
Usually it is not necessary for MLJBase or MLJ to be dependencies of a package implement the MLJ model interface. You just need MLJModelInterface.jl. For *testing* however, you will need to add `MLJBase` (or `MLJ`, or `MLJTestIntegration`). This will drop your dependency load considerably.